controllers/front/validation.php

```diff
@@ -145,7 +145,7 @@ public function postProcess()
             $mes = "Adr BTC : " . $address;
             $blockonomics->validateOrder(
                 (int) $cart->id,
-                Configuration::get('BLOCKONOMICS_ORDER_STATE_WAIT'),
+                (int) Configuration::get('PS_OS_PREPARATION'),
```

Comment:
PS_OS_PREPARATION (Processing in progress) refers to processing the order, which would happen after payment has been made. By default, this status marks an order as paid and should not be used.
![Statuses-Edit-Processing-in-progress-•-PrestaShop3 (1)](https://user-images.githubusercontent.com/39666372/116392314-96f52480-a820-11eb-9326-23d71cf04adc.png)

Our options to set the status to during validateOrder are:
- Awaiting check payment
- Awaiting bank wire payment
- On backorder (not paid)
- Awaiting Cash On Delivery validation

Reply:
Yes very good catch @DarrenWestwood , it is indeed marking the order as Paid. Unfortunately these order statuses are not suitable and more confusing.

Let us keep using Awaiting_Bitcoin_Payment Status. We will not delete it on module uninstallation

Reply:
Sounds good, I have reintroduced the Awaiting_Bitcoin_Payment, and it won't be removed in during the module uninstallation.